crates/storage/codecs/Cargo.toml

```diff
@@ -23,17 +23,18 @@ arbitrary = [
 [dependencies]
 bytes = "1.2.1"
 codecs-derive = { version = "0.1.0", path = "./derive", default-features = false }
-revm-interpreter = { git = "https://github.com/bluealloy/revm", rev = "a05fb262d87c78ee52d400e6c0f4708d4c527f32", features = [
-    "serde",
-] }
+revm-interpreter = {version = "1.0.0", features = ["serde"] }
+
+#     "serde",
+# ] }
```

Comment:
use revm_primitives instead, no need to have interpreter

Reply:
Fixed :-)